integration-tests/e2e/config/config.go

```diff
@@ -119,6 +122,9 @@ func init() {
 	if ShootKubeconfigPath == "" {
 		log.Fatal("shoot config not set")
 	}
+	GardenKubeconfigPath = os.Getenv("GARDEN_KUBECONFIG_PATH")
```

Comment:
I think we should also add commandline flags for theses values as it seems inconsistent that some input parameters can only be defined by env var and some by commandline